packages/protocol/contracts-0.8/common/ScoreManager.sol

```diff
@@ -52,7 +61,14 @@ contract ScoreManager is
     _transferOwnership(msg.sender);
   }
 
+  /**
+   * @notice Sets the group score for a specified group.
+   * @param group The address of the group wich score needs to be updated.
+   * @param score The new score of the group to be updated.
+   * @dev Set value to `ZERO_FIXED1_UINT` to set score to zero.
```

Comment:
That name is a bit confusing (has both 0 and 1), maybe something like `ZERO_SCORE` instead?